src/wallet/rpcdump.cpp

```diff
@@ -560,7 +560,7 @@ UniValue importwallet(const JSONRPCRequest& request)
 
         // Use uiInterface.ShowProgress instead of pwallet.ShowProgress because pwallet.ShowProgress has a cancel button tied to AbortRescan which
         // we don't want for this progress bar showing the import progress. uiInterface.ShowProgress does not have a cancel button.
-        uiInterface.ShowProgress(_("Importing..."), 0, false); // show progress dialog in GUI
+        uiInterface.ShowProgress(strprintf(_("%s Importing..."), pwallet->GetDisplayName()), 0, false); // show progress dialog in GUI
```

Comment:
If you prefix this, it would be better to not change the translation string. I.e. `strprintf("%s " + _("Imp...`

Reply:
agree